flac.go

```diff
@@ -134,6 +133,8 @@ var (
 	// id3Signature marks the beginning of an ID3 stream, used to skip over ID3 data.
 	id3Signature = []byte("ID3")
 
+	// ErrNoSeeker tells you if the Reader you passed to New/Parse is not
```

Comment:
I know we're not in academia so take this note as a personal preference, simple observation. Most documentation I've read has been in passive tone, so reading the active version of the doc comment made it stick out.

Posting a passive tone version of the error doc comment, just for comparison. I'm fine with using either one.

```go
// ErrNoSeeker reports that flac.NewSeek was called with an io.Reader not
// implementing io.Seeker, and thus does not allow for seeking.
```